src/mina-signer/src/sign-zkapp-command.unit-test.ts

```diff
@@ -82,42 +82,38 @@ expect(stringify(dummyInput.packed)).toEqual(
   stringify(dummyInputSnarky.packed)
 );
 
-test(Random.accountUpdate, (accountUpdate) => {
-  const testnetMinaInstance = Network({
-    networkId: 'testnet',
-    mina: 'http://localhost:8080/graphql',
-  });
-  const mainnetMinaInstance = Network({
-    networkId: 'mainnet',
-    mina: 'http://localhost:8080/graphql',
-  });
-
-  fixVerificationKey(accountUpdate);
-
-  // example account update
-  let accountUpdateJson: Json.AccountUpdate =
-    AccountUpdate.toJSON(accountUpdate);
-
-  // account update hash
-  let accountUpdateSnarky = AccountUpdateSnarky.fromJSON(accountUpdateJson);
-  let inputSnarky = TypesSnarky.AccountUpdate.toInput(accountUpdateSnarky);
-  let input = AccountUpdate.toInput(accountUpdate);
-  expect(toJSON(input.fields)).toEqual(toJSON(inputSnarky.fields));
-  expect(toJSON(input.packed)).toEqual(toJSON(inputSnarky.packed));
-
-  let packed = packToFields(input);
-  let packedSnarky = packToFieldsSnarky(inputSnarky);
-  expect(toJSON(packed)).toEqual(toJSON(packedSnarky));
-
-  let hashTestnet = accountUpdateHash(accountUpdate, 'testnet');
-  let hashMainnet = accountUpdateHash(accountUpdate, 'mainnet');
-  setActiveInstance(testnetMinaInstance);
-  let hashSnarkyTestnet = accountUpdateSnarky.hash();
-  setActiveInstance(mainnetMinaInstance);
-  let hashSnarkyMainnet = accountUpdateSnarky.hash();
-  expect(hashTestnet).toEqual(hashSnarkyTestnet.toBigInt());
-  expect(hashMainnet).toEqual(hashSnarkyMainnet.toBigInt());
-});
+test(
```

Comment:
@Trivo25 
Throughout different tests we check that signatures match like `testnet <-> testnet` but should we also check that the resulting signatures are indeed different? Like `testnet <-> mainnet` should always differ.

Reply:
Yes, we should! And we do
For example, further down we verify against a different network and expect the verification to fail
```ts
expect(
   verify(sigFieldElements, networkId === 'mainnet' ? 'testnet' : 'mainnet')
).toEqual(false);
```

Reply:
but I added a check that the hash doesnt match another networks hash!